packages/odyssey-react/src/components/Field/Field.module.scss

```diff
@@ -37,4 +37,6 @@
 
 .error {
   margin-block-start: $spacing-xs;
+  color: $color-danger-base;
+  font-size: $size-body-caption;
 }
```

Comment:
I believe you need a rebase here to remove this now that you've merged your other branch